lib/tbot/workloadidentity/trust_bundle_cache.go

```diff
@@ -203,6 +205,56 @@ type TrustBundleCacheConfig struct {
 	StatusReporter     readyz.Reporter
 }
 
+type TrustBundleCacheFacade struct {
```

Comment:
Is it worth having a short GoDoc here to explain that this Facade exists to provide lazy initialisation ?